ERCS/erc-7558.md

```diff
@@ -0,0 +1,69 @@
+---
+eip: 7558
+title: Minimal Lockable Range NFTs
+description: A minimal interface for soulbinding multiple ERC-721 NFTs
```

Comment:
Including "minimal" here in your description doesn't really add a lot of information, nor would it help to distinguish this proposal from hypothetical others in a similar theme. Try to make your description highlight what makes this proposal special/interesting.